includes/wf_crm_webform_postprocess.inc

```diff
@@ -1978,7 +1978,7 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
       }
     }
 
-    $params = $this->contributionParams();
+    $params = $this->contributionParams(FALSE);
```

Comment:
If we have default value FALSE for the method `contributionParams($isLivePayment = FALSE)`, no need to add bool FALSE here.